Source/WebCore/html/HTMLAttachmentElement.h

```diff
@@ -60,8 +59,10 @@ class HTMLAttachmentElement final : public HTMLElement {
 
     WEBCORE_EXPORT void updateAttributes(std::optional<uint64_t>&& newFileSize, const AtomString& newContentType, const AtomString& newFilename);
     WEBCORE_EXPORT void updateEnclosingImageWithData(const String& contentType, Ref<FragmentedSharedBuffer>&& data);
-    WEBCORE_EXPORT void updateThumbnail(const RefPtr<Image>& thumbnail);
-    WEBCORE_EXPORT void updateIcon(const RefPtr<Image>& icon, const WebCore::FloatSize&);
+    WEBCORE_EXPORT void updateNarrowThumbnail(const RefPtr<Image>& thumbnail);
+    WEBCORE_EXPORT void updateWideThumbnail(Vector<uint8_t>&&);
+    WEBCORE_EXPORT void updateNarrowIcon(const RefPtr<Image>& icon, const WebCore::FloatSize&);
+    WEBCORE_EXPORT void updateWideIcon(Vector<uint8_t>&&);
```

Comment:
This is more of a nit, but I would prefer:

`update{Thumbnail/Image}For{Wide/Narrow}Layout`

The current names sort of imply a different meaning.